imgui-glfw/src/commonMain/kotlin/com/imgui/impl/ImGuiGlfw.kt

```diff
@@ -0,0 +1,442 @@
+package com.imgui.impl
+
+import com.imgui.*
+import com.kgl.core.Flag
+import com.kgl.glfw.*
+import io.ktor.utils.io.core.*
+
+private var _mainWindow: Window? = null
+internal val mainWindow: Window get() = _mainWindow ?: error("ImGuiGLFW was not initialized")
+
+private val mouseJustPressed = BooleanArray(5) { false }
+private var wantUpdateMonitors: Boolean = true
+
+private var prevUserCallbackMouseButton: MouseButtonCallback? = null
+private var prevUserCallbackScroll: ScrollCallback? = null
+private var prevUserCallbackKey: KeyCallback? = null
+private var prevUserCallbackChar: CharCallback? = null
+private var prevUserCallbackMonitor: MonitorCallback? = null
+
+class ImGuiGlfw(window: Window, installCallbacks: Boolean) : Closeable {
```

Comment:
I mostly agree with the rename of `ImGuiGLFW` to `ImGuiGlfw` but it should be done separately in master first. Especially since Git can't seem to tell that it's a renamed file, instead delete+new.